### PR TITLE
[16.0][MIG] stock_landed_costs_purchase_auto: Migration to 16.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 16.0

Please @pedrobaeza and @CarlosRoca13 can you review it?

@Tecnativa TT47863

### CODE REVIEW

File: stock_landed_costs_purchase_auto/views/purchase_order_view.xml [hunk 1]
```diff
@@ -14,13 +13,19 @@
                     class="oe_stat_button"
                     icon="fa-list-alt"
                     attrs="{'invisible': [('landed_cost_number', '=', 0)]}"
+                    groups="stock.group_stock_manager"
                 >
                     <field
                         name="landed_cost_number"
                         widget="statinfo"
                         string="Landed Costs"
+                        groups="stock.group_stock_manager"
```
Comment (pedrobaeza) [inline stock_landed_costs_purchase_auto/views/purchase_order_view.xml]: No need of the group in nested elements.

File: stock_landed_costs_purchase_auto/views/purchase_order_view.xml [hunk 2]
```diff
@@ -14,13 +13,19 @@
...
+                    />
+                    <field
+                        name="landed_cost_ids"
+                        groups="stock.group_stock_manager"
```
Comment (pedrobaeza) [inline stock_landed_costs_purchase_auto/views/purchase_order_view.xml]: No need in nested elements.

**Review by pedrobaeza — APPROVED** [review]

**Review by CarlosRoca13 — APPROVED** [review]

**Pull request merged**